bin/get-json-config.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
```

Comment:
I think we need to be careful to either keep this tool separated from this one:

https://github.com/perftool-incubator/toolbox/blob/main/bin/get-json-settings.pl
https://github.com/perftool-incubator/toolbox/blob/main/perl/toolbox/jsonsettings.pm

Or figure out how to combine them.

The existing tool is used to process a file like this:

https://github.com/perftool-incubator/rickshaw/blob/master/rickshaw-settings.json

In ways such as these:

https://github.com/perftool-incubator/rickshaw/blob/master/rickshaw-run#L1393-L1441

https://github.com/perftool-incubator/rickshaw/blob/master/endpoints/base#L920-L958

https://github.com/perftool-incubator/rickshaw/blob/master/engine/engine-script-library#L1090-L1109

https://github.com/perftool-incubator/rickshaw/blob/master/engine/engine-script-library#L271

Reply:
I think we need to create either an examples directory or a doc directory, something like that.

Reply:
done